packages/test-middlewares/src/middlewares.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  in: [
+    async (context, next, done) => {
+      const muted = await context.brain.userGet(context.userMessage.user, '_isMuted');
+      if (muted) {
+        done();
+      } else {
+        next(done);
```

Comment:
@yangeorget Are there cases where we do not pass `done` to `next` ?
If not, would it be possible to automatically pass it in `botfuel-dialog` so we don't have to do it in bots?